### PR TITLE
Update the topology service documentation

### PR DESCRIPTION
**Pull request by frouioui**
This PR updates the topology service documentation.

### CODE REVIEW

File: content/en/docs/16.0/reference/features/topology-service.md [hunk 1]
```diff
@@ -108,21 +108,21 @@ The Tablet record has a lot of information about each vttablet process
 making up each tablet (along with the MySQL process):
```
Comment (rsajwani) [inline content/en/docs/16.0/reference/features/topology-service.md:107]: shard-global tablet controls, like denied tables no tablet should serve
  in this shard.
  >> something is missing in above sentence a comma or punctuation... i am giving it a try
shard-global tablet controls, like 'DeniedTables' which no tablet should serve
  in this shard.

File: content/en/docs/16.0/reference/features/topology-service.md [hunk 2]
```diff
@@ -108,21 +108,21 @@ The Tablet record has a lot of information about each vttablet process
...
```
Comment (rsajwani) [inline content/en/docs/16.0/reference/features/topology-service.md:108]: so multiple tasks cannot concurrently alter the data.

>> so multiple tasks cannot concurrently alter the same data.

File: content/en/docs/16.0/reference/features/topology-service.md [hunk 3]
```diff
@@ -143,7 +143,7 @@ It is the local representation of a Keyspace. It contains information on what
 shard to use for getting to the data (but not information about each individual
 shard):
 
-* the partitions map is keyed by the tablet type (primary, replica, batch, …) and
+* the partitions map is keyed by the Tablet type (primary, replica, batch, …) and
   the value is a list of shards to use for serving.
 * it also contains the global Keyspace fields, copied for fast access.
```
Comment (rsajwani) [inline content/en/docs/16.0/reference/features/topology-service.md:148]: It contains the VSchema for all keyspaces in a single object.
>> does that mean
It contains the VSchema for all keyspaces (in that cell) in a single object.

Comment (rsajwani) [inline content/en/docs/16.0/reference/features/topology-service.md:148]: would be good if you can link topocat here 
>> The `vtctl TopoCat` utility can decode these

Comment (rsajwani) [inline content/en/docs/16.0/reference/features/topology-service.md:148]: -tablet-path must be --tablet-path

Comment (rsajwani) [inline content/en/docs/16.0/reference/features/topology-service.md:148]: provide link of topo2topo

**Comment by frouioui** [comment]
@rsajwani, may i have a final review on this Pull Request? 🙏🏻

File: content/en/docs/16.0/reference/features/topology-service.md [hunk 4]
```diff
@@ -85,12 +85,12 @@ Global topology service contains:
 * if using filtered replication, the source shards this shard is replicating
   from.
 * the list of cells that have tablets in this shard.
-* shard-global tablet controls, like denied tables no tablet should serve
+* shard-global tablet controls, like `DeniedTable`s, which no tablet should serve
   in this shard.
```
Comment (rsajwani) [inline content/en/docs/16.0/reference/features/topology-service.md]: last try :) 

"shard-global tablet controls, like `DeniedTables`, which specifies a comma-separated list of tables to deny queries on."

**Review by rsajwani — APPROVED** [review]

**Pull request merged**